client/ctclient/ctclient.go

```diff
@@ -220,6 +220,13 @@ func getInclusionProof(ctx context.Context, logClient client.CheckLogClient) {
 			log.Fatalf("Failed to create hash of leaf: %v", err)
 		}
 		hash = leafHash[:]
+
+		// Print a warning if this timestamp is still within the MMD window
+		now := time.Now()
+		ts := time.Unix(0, *timestamp*1000000)
+		if age := now.Sub(ts); age < *logMMD {
```

Comment:
`age := time.Since(ts)` is recommended for anything like `time.Now().Sub(when)`.

Reply:
Ah, thanks - done